### ISSUE TITLE
#includes not being found VS 2017

### ISSUE THREAD

**Issue by xSeditx** [TheGameCreators/GameGuruRepo#471]
I am not sure if this is the place to ask this but has anyone else had a problem using this on VS 2017?
I have already gotten the project to compile fine in VS 2015 with almost no problems however moving the project to a different computer that is using VS 2017 I can not get any of the C++ library headers to be found. Stdio.h, Vector, Ctype.h to name a few... I have never come across such an issue so do not even know where to start. If I had to guess it is looking for Visual Studios 14.0 when my pathway is Visual Studios 14.1 but I have not confirmed this yet.

Does anyone know of a quick fix for this error?

**Comment by HierToo** [comment]
You need to set the Plattformtoolset to the VS2017 version instead of the VS2015(right click on the project and then choose settings). That should fix your problems.
If you see "Visual Studio 2015 (v140)", use "Visual Studio 2017 (v141)"
else "Visual Studio 2015 - Windows XP (v140_xp)" to "Visual Studio 2017 - Windows XP (v141_xp)"

**Comment by xSeditx** [comment]
When I cloned the project the first time I re-targeted the solution which changed the build tools because I did not even have build tools installed for 14.0 which I initially felt was the issue so I re-installed VS (repaired) as well as place new build tools explicitly following the instructions from how to build this project. I also during the modified install added the 14.0 build tools thinking that would solve the issues and re-cloned the project with the original settings onto a new drive thinking that should fix my issues since the first time I cloned this project on my laptop I was able to build and compile with no issues. It did not. 
Its about time for me to head back to that computer so I will give it another go and let you know what I find but I just find it all odd that not only have I gotten this to compile fine once but I moved other projects of mine from that same laptop over to this desktop and was able to get them to compile fine. 

I am trying not to alter the build tools(did not work when I initially tried it anyway) if possible because won't that possibly conflict if any changes are pushed? Sorry for my ignorance, never really tried to work on a project of this scale before.
